scripts/next-data/generateNodeReleasesData.mjs

```diff
@@ -0,0 +1,56 @@
+import nodevu from '@nodevu/core';
+import { writeFile } from 'fs/promises';
+import { join } from 'path';
+
+import { getRelativePath } from './_helpers.mjs';
+
+const __dirname = getRelativePath(import.meta.url);
+const jsonFilePath = join(
+  __dirname,
+  '../../public/static/node-releases-data.json'
+);
+
+const generateNodeReleasesData = async () => {
+  const [nodevuOutput, indexJsonOutput] = await Promise.all([
+    nodevu(),
+    // nodevu doesn't return release date and modules
+    // So it is a temporary workaround
+    (await fetch('https://nodejs.org/dist/index.json')).json(),
+  ]);
+
+  // Filter out those without documented support
+  // Basically those not in schedule.json
+  const majors = Object.values(nodevuOutput).filter(major => {
```

Comment:
```suggestion
  const majors = Object.values(nodevuOutput).filter(major => major?.support)
```

One liner simplified

Reply:
Noted, thanks!

Actually this line is gone after I removed the duplicate `v0.x` versions. But I'll keep this simplification in mind. 😄